### PR TITLE
use update stream instead of hardcode

### PR DESCRIPTION
**Pull request by sunafterrainwm**

### CODE REVIEW

**Review by ANPSC — APPROVED** [review]
Looks good for me.

**Comment by ANPSC** [comment]
Not working.

File: ericdress.py [hunk 1]
```diff
@@ -1,215 +1,49 @@
...
+		from .upstream_default import upstream_url
+
+ericliu_said = []
+
+async def init():
+	response = await client.get(upstream_url)
+	response.raise_for_status()
+	ericliu_said = str(response.text).split(",\n")
+
+@listener(command="updateericdress",
```
Comment (ANPSC) [inline ericdress.py:23]: 看起来不能和updatesteram配合使用

**Comment by ANPSC** [comment]
@ANPSC Test

**Comment by sunafterrainwm** [comment]
@ANPSC

**Pull request merged**